docs/faq/general/why-cant-i-see-my-organization.md

```diff
@@ -4,16 +4,18 @@ description: Try these steps if you don't see your organization when adding your
 
 # Why can't I see my organization?
 
-If you don't see your organization when [adding your organization on Codacy](../../organizations/what-are-synced-organizations.md#adding-an-organization), try the following steps:
+Follow the steps below if you don't see your organization while [adding your organization on Codacy](../../organizations/what-are-synced-organizations.md#adding-an-organization):
 
-1.  Make sure you have access to the organization with the account you're logged in.
+1.  Make sure that you have access to the organization on the Git provider using the account that you used to log in on Codacy.
 
 1.  **If you're using GitHub** [install and authorize Codacy on your organization](https://github.com/apps/codacy-production/installations/new).
 
-1.  Refresh the list of organizations on Codacy by clicking **refresh this list** on the Organizations page:
```

Comment:
I would keep this option separate from the other and appearing before. Its a separate way to make sure the org list is ok. 

also because after you revoke

Reply:
Yes, I see what you mean. This could be tried after each individual troubleshooting step. I'll make an experiment. :ok_hand:

Reply:
Please check the new version. :eyes: